### ISSUE TITLE
FEAT: MAKE NEWSLETTER FORM ACCESSIBLE 💡 

### ISSUE THREAD

**Issue by 0xabdulkhalid** [amanjaiman1/Product_3D#511]
### Is your feature request related to a problem? Please describe.

- This feature related to problems with accessibility standards.

### Describe the solution you'd like

- I would like to improve the newsletter form accessibility on the homepage by adding important label element with screen reader only class and associated styles for it.
- So it could perform best on accessibility cases.

### Describe alternatives you've considered

- There's no alternative for labels in terms of accessibility

## Screenshots (Going to make changes on these elements)
![Screenshot 2023-07-22 at 20-19-40 Fashion Forge 3D](https://github.com/amanjaiman1/Product_3D/assets/92252895/41d10a55-11b5-4740-a1ba-2e1c2ff92f12)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by 0xabdulkhalid** [comment]
Hey there @amanjaiman1, @balaji-sivasakthi 👋🏻

Please make sure to assign this issue to me as soon as possible so that i can proceed my progress regarding fixing this issue.

Thank you!
